lib/dispatcher/client-h2.js

```diff
@@ -87,12 +89,16 @@ function parseH2Headers (headers) {
 function connectH2 (client, socket) {
   client[kSocket] = socket
 
+  const http2InitialWindowSize = client[kHTTP2InitialWindowSize]
+  const http2ConnectionWindowSize = client[kHTTP2ConnectionWindowSize]
```

Comment:
I think we can have slightly better defaults than Node.js core for these.
See @ronag in https://github.com/nodejs/node/pull/61036